lusidtools/cocoon/utilities.py

```diff
@@ -76,11 +76,18 @@ def _f(*args, **kwargs):
                 else:
                     is_default_value = argument_value == argument_details.default
 
+            # Acceptable edge cases
+            # condition 1. check instance matches expected instance type
+            condition_1 = not isinstance(argument_value, argument_details.annotation)
+
+            # condition 2. check annotation is not empty
+            condition_2 = argument_details.annotation is not argument_details.empty
+
+            # condition 3. allow assignment of scope to None if file_type is instrument
+            condition_3 = not (kwargs.get("file_type") in ["Instruments", "instruments"] and argument_name == 'scope')
```

Comment:
Do we want to change this behaviour? What is the benefit of adding this? It hasn't been possible to specify this as None up until now, why should we allow it?